servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -477,7 +477,7 @@ void RenderForwardClustered::_render_list_template(RenderingDevice::DrawListID p
 			prev_material_uniform_set = material_uniform_set;
 		}
 
-		if (surf->owner->base_flags & INSTANCE_DATA_FLAG_MULTIMESH) {
+		if (surf->owner->base_flags & (INSTANCE_DATA_FLAG_MULTIMESH | INSTANCE_DATA_FLAG_PARTICLES) == INSTANCE_DATA_FLAG_MULTIMESH) {
```

Comment:
```suggestion
		if ((surf->owner->base_flags & (INSTANCE_DATA_FLAG_MULTIMESH | INSTANCE_DATA_FLAG_PARTICLES)) == INSTANCE_DATA_FLAG_MULTIMESH) {
```

CI is complaining of missing parentheses